src/backend/joanie/core/utils/contract_definition.py

```diff
@@ -104,16 +105,27 @@ def generate_document_context(contract_definition, user, order=None):
         try:
             organization_address = order.organization.addresses.get(is_main=True)
         except Address.DoesNotExist:
-            organization_address = ORGANIZATION_FALLBACK_ADDRESS
+            organization_address = None
```

Comment:
Why can't we keep the default fallback placeholder for the organization's address ?

IMO, if an organization has forgotten to set an address (somehow...), we should give them 
default placeholder values for the generation of the context for the template, the same way we(ve implemented the fallback values for the student's address.

Reply:
IMO this is weird to display a fallback in a real contract.